core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -360,7 +361,8 @@ describe('Creator dashboard controller', () => {
         expect(ctrl.myExplorationsView).toBe('a');
       });
 
-      it('should set explorations sorting options', function() {
+      it('should change explorations sorting options by number of open' +
```

Comment:
explorations --> exploration

Also this is not specific. Have your "when ..." term be really specific about the action that triggers the change, not focused around the "rough general thing that the user is doing". In this case, the trigger is that the "user re-clicks the same exploration sorting options" and the first check after the action checks that the sort order is flipped. The second trigger is that the "user selects a different sorting option" and the check is that the sort type is changed. This analysis suggests that this should actually be two different tests.

This is why naming is important. Please don't do it mechanically. Read the test and think through what each expectation (or set of expectations) is actually checking, then describe that precisely. If two checks end up being unrelated, then put them into separate tests.

Reply:
Done.